src/obp_accounting_sdk/_async/longrun.py

```diff
@@ -0,0 +1,239 @@
+"""Longrun session."""
+
+import logging
+import platform
+import signal
+import time
+from http import HTTPStatus
+from multiprocessing import Process, set_start_method
+from types import TracebackType
+from typing import Self
+from uuid import UUID
+
+import httpx
+
+from obp_accounting_sdk.constants import (
+    HEARTBEAT_INTERVAL,
+    LongrunStatus,
+    ServiceSubtype,
+    ServiceType,
+)
+from obp_accounting_sdk.errors import (
+    AccountingCancellationError,
+    AccountingReservationError,
+    AccountingUsageError,
+    InsufficientFundsError,
+)
+from obp_accounting_sdk.utils import get_current_timestamp
+
+L = logging.getLogger(__name__)
+
+
+if platform.system() != "Linux":
+    set_start_method("fork")
```

Comment:
I'm wondering why requiring fork instead of spawn? 
Note also that we shouldn't set this globally in a library, because the user code could have different needs. Also, from the docs:

> [set_start_method] should be called at most once, and it should be protected inside the `if __name__ == '__main__'` clause of the main module.

> [fork] is currently the default on POSIX except macOS. [...] The default start method will change away from fork in Python 3.14

Reply:
Didn't look deep enough to understand the root cause, but there is something with how the spawn works in macOS: it was just hanging the child process. I would leave it as is to unblock Dries for now. (I guess I'll create an issue for that)

Reply:
Would `spawn` currently work on linux, instead?
If not, it's safer to explicit set `fork` also in linux, or it will probably break in python 3.14 where the default start method will change away from fork.
It's also better to add a comment in the code with the reason for using fork.